Tests/LibWeb/Text/expected/css/getComputedStyle-print-all.txt

```diff
@@ -197,6 +197,7 @@ text-decoration-style: solid
 text-decoration-thickness: auto
 text-indent: 0px
 text-justify: auto
+text-overflow: clip
```

Comment:
Please move this change to where `text-overflow` was added, so that the test doesn't fail on the in-between commits. I'm not sure which commit that would be, possibly the code generation one?